code/modules/reagents/reagent_containers/food/snacks/cultural/human.dm

```diff
@@ -506,6 +538,13 @@
 			user.put_in_hands(waste)
 		qdel(src)
 
+/obj/item/reagent_containers/food/snacks/bowl/MouseDrop(mob/user) //Dropping the bowl of food onto the user
+	if(istype(user) && user == usr)
```

Comment:
`&& !user.use_check_and_message()`

Reply:
What? I don't know what that is or where to put that, this code is copied from the mini-avahs or bowl of peanuts code.